### PR TITLE
Adding norwegian CV template option

### PR DESCRIPTION
**Pull request by gmkvaal**
Added a flag "--norwegian" to the CLI which renders Norwegian titles, so that we can write CVs in Norwegian. 

This functionality can easily extend to other languages, in case one wants to write a CV Uzbek etc.

### CODE REVIEW

**Comment by bzapf** [comment]
Thanks, works nicely!

File: cvcreator/vitae/schema.py [hunk 1]
```diff
@@ -161,6 +161,49 @@ class MetaInformation(StrictModel):
...
+class SectionTitles(StrictModel):
+    """Titles of the different sections. In English by default."""
+
+    professional_experience: str = "Professional Experience"
+    education: str = "Education"
+    technical_skills: str = "Technical Skills"
+    languages: str = "Languages"
+    personal_skills: str = "Personal Skills"
+    hobbies: str = "Interests and hobbies"
+    projects: str = "Extended description of selected projects"
```
Comment (alexanfl) [inline cvcreator/vitae/schema.py:173]: Suggestion to have capitalization in these titles as well

```suggestion
    hobbies: str = "Interests and Hobbies"
    projects: str = "Extended Description of Selected Projects"
```

**Review by alexanfl — APPROVED** [review]
LGTM, just one minor suggestion

**Comment by alexanfl** [comment]
Need to update `version` in `pyproject.toml` as well, perhaps?

**Comment by bzapf** [comment]
The accepted values should probably also be changed?
For example, 
`  unexpected value; permitted: 'Native', 'Fluent', 'Intermediate', 'Basic' (type=value_error.const; given=Flytende; permitted=('Native', 'Fluent', 'Intermediate', 'Basic'))`
same for, e.g., the degree fields. 
I can fix that, is it better to create a new PR or push to this branch?

**Pull request merged**